tests/items/test_ratings.py

```diff
@@ -0,0 +1,124 @@
+"""Tests for Ratings."""
+
+from typing import Dict, List
+
+import pytest
+from dialoguekit.core.domain import Domain
+
+from usersimcrs.items.item_collection import ItemCollection
+from usersimcrs.items.ratings import Ratings, user_item_sampler
+
+DOMAIN_YAML_FILE = "tests/data/domains/movies.yaml"
+DOMAIN_MAPPING = {
+    "title": {"slot": "TITLE"},
+    "genres": {
+        "slot": "GENRE",
+        "multi-valued": True,
+        "delimiter": "|",
+    },
+}
+ITEMS_CSV_FILE = "tests/data/items/movies.csv"
+RATINGS_CSV_FILE = "tests/data/items/ratings.csv"
+
+
+def simple_user_item_sampler(
+    item_ratings: Dict[str, float],
+    historical_ratio: float,
+) -> List[str]:
+    """Creates a sample of items for a given user.
```

Comment:
Nit: Can you mention that it takes the first portion of ratings as opposed to a random sample?